OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -115,12 +115,14 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
 // OUTPUTS
 //=============================================================================
     OpenSim_DECLARE_OUTPUT(power, double, calcPower, SimTK::Stage::Acceleration);
-    // reaction_load_at_parent_frame
-    OpenSim_DECLARE_OUTPUT(reaction_load_at_parent_frame_in_ground_frame, SimTK::SpatialVec,
-        calcReactionAtParentFrameInGround, SimTK::Stage::Acceleration);
-    // reaction_load_at_child_frame
-    OpenSim_DECLARE_OUTPUT(reaction_load_at_child_frame_in_ground_frame, SimTK::SpatialVec,
-        calcReactionAtChildFrameInGround, SimTK::Stage::Acceleration);
+    /** get the joint reaction force and moment experienced on the parent frame
+        and expressed in ground. */
+    OpenSim_DECLARE_OUTPUT(reaction_on_parent, SimTK::SpatialVec,
+        calcReactionOnParentExpressedInGround, SimTK::Stage::Acceleration);
+    /** alternatively, get the joint reaction force and moment experienced on
+        the child frame and expressed in ground. */
+    OpenSim_DECLARE_OUTPUT(reaction_on_child, SimTK::SpatialVec,
+        calcReactionOnChildExpressedInGround, SimTK::Stage::Acceleration);
```

Comment:
Good point. I think  the method documentation is already better anyways. I will clear it out.